### ISSUE TITLE
OrbitPlot sizing problem

### ISSUE THREAD

**Issue by dtamayo** [hannorein/rebound#417]
It seems like OrbitPlot has been refactored in a nice way to give the user more control, but here's a simple case that probably should give a reasonable plot out of the box, but gives a very squeezed plot

```
sim = rebound.Simulation()
sim.add(m=1)
sim.add(a=1)
fig,ax = rebound.OrbitPlot(sim)
```

It seems like now we are letting matplotlib do the limits based on what is plotted. For some reason this works OK for more than 1 planet, but fails for this one. Would we want to go back to assigning a default limit based on semimajor axis?

**Comment by hannorein** [comment]
It's really tricky to make the automatic limits work in all cases. The other problems that I had involved very eccentric particles and plots with `slices=True`. I found that in particular the `slices` option is almost impossible to behave in most cases (It works reasonably well right now, even for the above example). So given that the limits can be easily adjusted, I would opt for not changing anything.

```
sim = rebound.Simulation()
sim.add(m=1)
sim.add(a=1.)
fig,ax = rebound.OrbitPlot(sim)
ax.set_xlim([-1,1])
ax.set_ylim([-1,1])
```
